examples/cars/gen/http/openapi.yaml

```diff
@@ -0,0 +1,115 @@
+swagger: "2.0"
+info:
+  title: Cars Service
+  description: HTTP service to lookup car models by body style.
+  version: ""
+host: localhost:8080
+paths:
+  /cars:
+    get:
+      tags:
+      - cars
+      summary: list cars
+      description: |-
+        Lists car models by body style.
+
+        Required security scopes:
+          * `stream:read`
+      operationId: cars#list
+      parameters:
+      - name: style
+        in: query
+        description: The car body style.
+        required: true
+        type: string
+        enum:
+        - sedan
+        - hatchback
+      responses:
+        "200":
```

Comment:
That doesn't seem right? seems like it should be `101`? Is there a way to represent websocket in swagger v2?

Reply:
There is no way to represent websocket in v2.